audit_exceptions/unstable_allowlist.json

```
@@ -3,6 +3,7 @@
   "automysqlbackup": "3.0-rc",
   "aview": "1.3.0rc",
   "ftgl": "2.1.3-rc",
+  "go": "1.18rc1",
```

```suggestion
  "go": "1.18rc",
```

There's a bug in `FormulaAuditor#audit_specs` that will cause the tap-syntax step to fail on CI despite the `unstable_allowlist.json` addition but we can work around it for the time being using this modified version string. This change won't be merged into master in the end, so it's fine for now to allow this to move forward in the interim time.